SYCL/XPTI/basic_event_collection.cpp

```diff
@@ -3,6 +3,8 @@
 // RUN: %clangxx -fsycl -fsycl-unnamed-lambda -fsycl-targets=%sycl_triple %s -o %t.out
 // RUN: env XPTI_TRACE_ENABLE=1 env XPTI_FRAMEWORK_DISPATCHER=%xptifw_dispatcher env XPTI_SUBSCRIBERS=%t_collector.dll env SYCL_DEVICE_FILTER=opencl %t.out | FileCheck %s 2>&1
 
+// It looks like order of events diffres on Windows
+// XFAIL: windows
```

Comment:
```suggestion
// REQUIRES: TEMPORARY_DISABLED
```
This test failed in pre-commit for this PR on Linux(!).